PyMca5/PyMcaGui/pymca/McaWindow.py

```diff
@@ -47,13 +47,13 @@
 
 from PyMca5.PyMcaGui.io import PyMcaFileDialogs
 from PyMca5.PyMcaGui.plotting.PyMca_Icons import IconDict
-from .ScanWindow import ScanWindow
-from . import McaCalibrationControlGUI
+from PyMca5.PyMcaGui.pymca import ScanWindow
+from PyMca5.PyMcaGui.pymca import McaCalibrationControlGUI
```

Comment:
In the spirit of minimizing changes to the upstream source code.

```suggestion
from PyMca5.PyMcaGui.pymca import McaCalibrationControlGUI
from PyMca5.PyMcaGui.pymca.ScanWindow import ScanWindow
```